packages/node_modules/@webex/internal-plugin-device/src/features/feature-model.js

```diff
@@ -79,6 +79,57 @@ const FeatureModel = AmpState.extend({
 
   // Ampsersand method members.
 
+  /**
+   * Parse all attribute values and map them to their appropriate members.
+   *
+   * @param {FeatureModel} pto - Attributes to be parsed and mapped.
+   * @returns {FeatureModel} - Parsed and mapped attributes.
+   */
+  parse(pto) {
+    // Generate a return transfer object.
+    let rto = {};
+
+    // Validate that a parameter transfer object was provided.
+    if (!pto) {
+      return rto;
+    }
+
+    // Clone the parameter transfer object into the return transfer object.
+    rto = {...pto};
```

Comment:
If you're going to clone it like this, then the above check to see if `pto` is defined is not necessary.
Cloning in line 90 like so `const rto = {..pto};` would create an empty object if there's nothing in `pto`.

Reply:
I will add a default value of `{}` for the `pto` object.

Reply:
Even if `pto` is undefined, `{..pto}` would still give you `{}` so no need to add the default value. Although, I don't think it can hurt